crates/common/src/fork.rs

```diff
@@ -54,6 +62,62 @@ pub async fn fork_call<'a>(
     }
 }
 
+/// checks the front matter validity and returns the deployer address if check went ok
+pub async fn fork_parse_dotrain(
+    rainlang_string: &str,
+    front_matter: &str,
+    fork_url: &str,
+    fork_block_number: u64,
+) -> Result<Bytes, String> {
```

Comment:
can this return the actual error instead of a string?

Reply:
sure